### PR TITLE
FEAT: Support XINFERENCE_DISABLE_METRICS env

### PR DESCRIPTION
**Pull request by codingl2k1**
Fixes: https://github.com/xorbitsai/inference/issues/1492

### CODE REVIEW

**Review by qinxuye — COMMENTED** [review]
Can you add a doc to https://github.com/xorbitsai/inference/blob/main/doc/source/getting_started/environments.rst?

**Comment by codingl2k1** [comment]
> https://github.com/xorbitsai/inference/blob/main/doc/source/getting_started/environments.rst

Added the doc for this env. Thanks.

File: doc/source/getting_started/environments.rst [hunk 1]
```diff
@@ -39,4 +39,10 @@ Setting this environment to 1 can disable health check.
 XINFERENCE_DISABLE_VLLM
 ~~~~~~~~~~~~~~~~~~~~~~~~
 Xinference will automatically use vLLM as backend if conditions are met.
-Setting this environment to 1 can disable the use of vLLM.
\ No newline at end of file
+Setting this environment to 1 can disable the use of vLLM.
+
+XINFERENCE_DISABLE_METRICS
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+Xinference will disable the metrics exporter on the supervisor and worker.
```
Comment (qinxuye) [inline doc/source/getting_started/environments.rst:46]:
```suggestion
Xinference will by default enable the metrics exporter on the supervisor and worker.
```

**Review by qinxuye — APPROVED** [review]
LGTM

**Pull request merged**